glean-core/android/src/test/java/mozilla/telemetry/glean/TestUtil.kt

```diff
@@ -256,3 +256,39 @@ fun RecordedRequest.getPlainBody(): String {
         this.body.readUtf8()
     }
 }
+
+// The following Mockito fixups are copied over from support-test (Matchers.kt) from
+// Android-Components. We copied them over since A-C uses the Glean SDK, and we don't
+// want a dependency on A-C.
+
+/**
+ * Mockito matcher that matches <strong>anything</strong>, including nulls and varargs.
+ *
+ * (The version from Mockito doesn't work correctly with Kotlin code.)
+ */
+internal fun <T> any(): T {
+    Mockito.any<T>()
+    return uninitialized()
```

Comment:
that is a weird looking function definition.

Reply:
Indeed!